Overview.html

```diff
@@ -1829,7 +1834,7 @@ <h2 id="fetching"><span class="secno">5 </span>Fetching</h2>
 <h3 id="main-fetch"><span class="secno">5.1 </span>Main fetch</h3>
 
 <p>To perform a <dfn id="concept-main-fetch" title="concept-main-fetch">main fetch</dfn> using
-<var>request</var>, optionally with a <i title="">CORS flag</i>, run these steps:
+<var>request</var>, with an optional <i title="">CORS</i> and <i title="recursive-flag">recursive flag</i>, run these steps:
```

Comment:
Please keep "CORS flag" as a single term. See e.g., https://fetch.spec.whatwg.org/#concept-http-network-or-cache-fetch for how this is worded elsewhere.